docs/modules/ROOT/pages/apdx-reactorExtra.adoc

```diff
@@ -45,7 +45,7 @@ You can rewrite the preceding example as follows:
 [source,java]
 [%unbreakable]
 ----
-.map(TupleUtils.function(Customer::new)); // <1>
+.map(TupleUtils.Function3(Customer::new)); // <1>
```

Comment:
I don't see `TupleUtils.Function3` anywhere, can you provide a code example demonstrating what you came up with?